### PR TITLE
External sqlite

### PR DESCRIPTION
**Pull request by milgner**
It seems to work now - check it out...

### CODE REVIEW

File: SQLite3/SQLite3.vcxproj [hunk 1]
```diff
@@ -41,17 +38,19 @@
...
   </ItemGroup>
+  <ItemGroup>
+    <SDKReference Include="SQLite.WinRT, Version=3.7.14" />
+  </ItemGroup>
   <PropertyGroup Label="Globals">
     <VCTargetsPath Condition="'$(VCTargetsPath11)' != '' and '$(VSVersion)' == '' and '$(VisualStudioVersion)' == ''">$(VCTargetsPath11)</VCTargetsPath>
     <ProjectGuid>{4cf1ae34-d183-409b-90c3-d62167cff25d}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
-    <ProjectName>SQLite3</ProjectName>
+    <ProjectName>SQLite3Wrapper</ProjectName>
```
Comment (pke) [inline SQLite3/SQLite3.vcxproj]: Shouldn't we name it "SQLite3 Component" as in "WinRT Component"?

Comment (milgner) [inline SQLite3/SQLite3.vcxproj]: But the library itself is a component now, too. And since the purpose of this project is to wrap it for use in JS...

Comment (milgner) [inline SQLite3/SQLite3.vcxproj]: So I just had a look and now I'm not sure whether the other project is a component, really... Need to check again later....

Comment (pke) [inline SQLite3/SQLite3.vcxproj]: Really? I thought its merely a WinRT API compatible version of the basic sqlite API. Not a component. A component consist of a dll and a md file.

Comment (milgner) [inline SQLite3/SQLite3.vcxproj]: True. Will do a rename...

**Comment by milgner** [comment]
Here we go :)

**Pull request merged**